VERSION-RANGE-SPEC.rst

```diff
@@ -0,0 +1,658 @@
+======================================================
+vers: a mostly universal version range specifier
```

Comment:
nit: People will pronounce this. Is it "verssss", "verz", "versh" (as in version), etc..

Reply:
> nit: People will pronounce this. Is it "verssss", "verz", "versh" (as in version), etc..

What do you prefer? :angel:

Reply:
I will say "versh"!